docs/content/integrations/dbt/reference.mdx

```diff
@@ -334,6 +335,58 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
     yield from dbt.cli(["build"], context=context).stream()
 ```
 
+### Customizing owners
+
+For dbt models, seeds, and snapshots, the default Dagster owner will be the email associated with the [dbt group](https://docs.getdbt.com/docs/build/groups) defined for that node.
+
+| dbt node type         | dbt group name | dbt group's email   | Resulting Dagster owner |
+| --------------------- | -------------- | ------------------- | ----------------------- |
+| model, seed, snapshot | `GROUP_NAME`   | `OWNER@DOMAIN.COM`  | `OWNER@DOMAIN.COM`      |
```

Comment:
What's the difference between the first row and the fourth row here?

Also, entries in the other columns are duplicated across rows, so maybe the entry in the first column should also be.

Reply:
Ah, I followed the same pattern I did for the other references. The first few rows are variables (which is why they're capitalized and in variable notation), and then the subsequent rows are examples with actual values (e.g. `finance`, and `owner@company.com`)

I could make the variable notation more explicit by using brackets (e.g. `<GROUP_NAME>`).

Reply:
Got it - hadn't noticed that before. I think this pattern is a little weird because it mixes levels of abstraction in the same table.

Not a blocker for this PR though.

Reply:
Yeah you're right. I can change this later.